internal/cmd/kafka/command_consumer.go

```diff
@@ -0,0 +1,54 @@
+package kafka
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
+)
+
+type consumerCommand struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+type consumerOut struct {
+	ConsumerGroupId string `human:"Consumer Group" serialized:"consumer_group"`
+	ConsumerId      string `human:"Consumer" serialized:"consumer"`
+	InstanceId      string `human:"Instance" serialized:"instance"`
+	ClientId        string `human:"Client" serialized:"client"`
```

Comment:
I think across the board, we suffix the serialized field name with "_id" when the struct field name ends with "Id"